src/test/java/seedu/taskify/model/util/SampleDataUtilTest.java

```diff
@@ -0,0 +1,44 @@
+package seedu.taskify.model.util;
+
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static seedu.taskify.testutil.TypicalTasks.TASK_1;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import seedu.taskify.model.ReadOnlyTaskify;
+import seedu.taskify.model.task.Task;
+import seedu.taskify.testutil.TaskBuilder;
+
+class SampleDataUtilTest {
+
+    @Test
+    public void getSampleTaskifyData_equalsDifferentTaskify_fail() {
+        Task editedSampleTask = new TaskBuilder(TASK_1).withName("test").build();
+        List<Task> sampleTasks = Arrays.asList(TASK_1, editedSampleTask);
+        TaskifyStub sampleTaskify = new TaskifyStub(sampleTasks);
+
+        assertNotEquals(sampleTaskify, SampleDataUtil.getSampleTaskifyData());
+    }
+
```

Comment:
I thought line 26 would automatically fail regardless of whether the tasks in the two arguments of `assertNotEquals` are the same because ReadOnlyTaskify doesn't have an overridden `equals`?

Reply:
Taskify.java has an equals() method that checks if the list of `tasks` is equal to the other Taskify object's `tasks` right?

Reply:
Ah nevermind, I compared the ReadOnlyTaskify instead of comparing the ReadOnlyTaskify#getTaskList(), I pushed a new commit